api/users/serializers.py

```diff
@@ -1,6 +1,87 @@
 from rest_framework import serializers as ser
 from api.base.serializers import JSONAPISerializer, LinksField, Link
 from website.models import User
+from rest_framework.fields import empty, SkipField, get_attribute, CharField, MaxLengthValidator, MinLengthValidator, URLValidator, URLField
+
+
+class SocialCharField(ser.CharField):
```

Comment:
Let's name this a bit more generically. Something like `CharFieldWithReadDefault` and change the variable to match. We don't need to go too crazy with the descriptive name, but put in the docstring what the purpose of this field is (i.e. handle if a dictionary is being accessed for the field but the field needs to return a default value).